Editors/PathEditor.cpp

```diff
@@ -234,15 +238,15 @@ void PathEditor::on_deletePointButton_pressed() {
   int deleteIndex = _ui->pointsTableView->selectionModel()->currentIndex().row();
   {
     RepeatedMessageModel::RowRemovalOperation remover(_pointsModel);

```

Comment:
Thought this was going to be a for-loop, now. Write it like this, please:

```cpp
RepeatedMessageModel::RowRemovalOperation(_pointsModel)
    .RemoveRows(_ui->pointsTableView->selectionModel()->selectedRows());

Reply:
Aight 01f9cdd48bdbd49cb590d7da9caceff4d97d407c and it is a for loop, it's a for loop in the class so we don't duplicate it everywhere.
https://github.com/enigma-dev/RadialGM/pull/179/files#diff-9223a9db2dfd4809200a0a8574385ed1R147